internal/lagoon/tasks.go

```diff
@@ -112,7 +112,11 @@ func getConfig() (*rest.Config, error) {
 // ExecuteTaskInEnvironment .
 func ExecuteTaskInEnvironment(task Task) error {
 	if debug {
-		fmt.Printf("Executing task '%v':'%v'\n", task.Name, task.Command)
+		podContainerDetails := fmt.Sprintf("Service:%v", task.Service)
```

Comment:
maybe lower case this for consistency?

```suggestion
		podContainerDetails := fmt.Sprintf("service:%v", task.Service)
```